x-pack/plugins/enterprise_search/public/applications/applications/components/search_application/connect/search_application_api_integration.tsx

```diff
@@ -244,10 +232,10 @@ export const SearchApplicationApiIntegrationStage: React.FC = () => {
             <EuiText>
               <FormattedMessage
                 id="xpack.enterpriseSearch.searchApplications.searchApplication.searchApi.step4.clientUsageDescription"
-                defaultMessage="To get the most out of the client, use the javascript client's example template and follow our {searchapplicationGettingStartedDocLink} on building a search experience."
+                defaultMessage="To get the most out of the client, use the javascript client's example template and follow our {searchapplicationSearchDocLink} on building a search experience."
```

Comment:
```suggestion
                defaultMessage="To get the most out of the JavaScript client, use the client's example template and follow our {searchapplicationSearchDocLink} on building a search experience."
```

Minor copyedit while we're here :)

Reply:
Added in 73065d895b67f342c6002fd7057a881087b866ef